### DISCUSSION TITLE
Definicion del Proyecto imposible

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-Electronica-Microcontrolada/Grupo4#61] [Announcements]
Que tal estimados, les dejo este espacio para que definan su proyecto y lo expliquen brevemente. De esta forma podremos habrir el registro con el profesor Jorge. Haganlo con el siguiente formato: 

Tema del Proyecto: 
Nombre del Proyecto:  
Definicion del Proyecto:

**Comment by JorEl057** [comment]
Chicos, que tal como estan :

 Manejo de Control de versiones: Ok
 Manejo de Estructura de proyecto: Ok
 Presentación Final: Falta presentar video con la demostración de lo hecho, la presentación grupal, subir código.

**Comment by JorEl057** [comment]
Devolución Grupal: @ivanexegomez  @matiluajn  @Daninja111  @alfredop37  @maxii-sc  @guelo2019 
Nota Final Grupal: 7 (Siete)

Ideación del Proyecto: 7 (siete)
Presentación: 6 (Seis)
Uso de recursos: 10 (diez)
Desarrollo: 6 (Seis)
Implementación: 6 (Seis)

Nota Final Grupal: 7(siete)

Estimados tomaron la idea de Semaforización Inteligente como proyecto y la ampliaron como un proyecto general de inteligencia y comunicación entre los semáforos sobre regiones con tráfico de alta densidad y complicado. Esto demuestra su compromiso y RSU. En su presentación que cumplieron correctamente con un tiempo acorde, hablan sobre la necesidad en forma general y la implementación en forma particular. Trabajaron demostrando una organización y de recursos agiles muy bien logrados. Finalmente el desarrollo y la implementación del proyecto también fueron logrados. Felicitaciones y continúen trabajando así!!!
